src/sql/workbench/parts/jobManagement/electron-browser/media/jobHistory.css

```diff
@@ -179,7 +179,7 @@ table.step-list tr.step-row td {
 }
 
 .history-details {
-	flex: 1 1 auto;
+	height: 100%;
```

Comment:
as we chatted offline, the issue here is the container of this element is no longer a flex container, should fix on the other side.